### PR TITLE
Fix typos. Improve language.

### PR DESCRIPTION
**Pull request by eriksjolund**
Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
/approve
LGTM
How are you finding these?  Are you using some tool other then codespell?

File: libpod/runtime.go
Comment (edsantiago) [inline libpod/runtime.go:708]: Might as well fix this one ("gourtine") too, no? How did it slip past? (insert comment about French-Canadian glopmixes of frites and gravy).

FWIW I think the sentence would read better as "...spawns a goroutine which will listen for events in the background..." (or even skipping the "in the background", since that feels implicit).

File: libpod/volume.go
Comment (edsantiago) [inline libpod/volume.go:74]: No, this comma, is, not desired.

File: test/python/docker/compat/common.py
Comment (edsantiago) [inline test/python/docker/compat/common.py:2]: Python expert wanted here: I believe in Pythonland it is idiomatic to write it as one word, because that's the name of the module. This needs review by someone proficient in Pythonspeak.

**Review by edsantiago — CHANGES_REQUESTED** [review]
ALmost LGTM. Nice work, thank you!

File: libpod/volume.go
Comment (eriksjolund) [reply libpod/volume.go]: You're right. I removed the comma

File: test/python/docker/compat/common.py
Comment (eriksjolund) [reply test/python/docker/compat/common.py]: I reverted the change as I think you have a valid point about spelling in Pythonland.

File: libpod/runtime.go
Comment (eriksjolund) [reply libpod/runtime.go]: I tried to change according to your suggestion. Does it look ok now?

File: test/python/docker/compat/constant.py
Comment (edsantiago) [inline test/python/docker/compat/constant.py:2]: Looks good but I should've pointed out this other python reference to one-word unittests. Could you do a quick repush? (There's one other "unittests", above, but that's Go and I think your edit is correct). Sorry for not pointing this one out.

**Review by edsantiago — APPROVED** [review]
LGTM, thank you!

**Comment by edsantiago** [comment]
@containers/podman-maintainers PTAL.

Note that the `api change`  label was set automatically by a bot because of files touched under `pkg/api`. Most of the changes are innocuous, but there [is one changed error message](https://github.com/containers/podman/pull/17459/files#diff-14dbec4a3c47cef4d74aa262b5aaebe42a88e7f91c284e58266df14e36220a47). I'm erring on the side of caution, and leaving the label.

**Review by vrothberg — COMMENTED** [review]
LGTM, thank you!

**Pull request merged**
